src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.tsx

```diff
@@ -170,11 +167,7 @@ const MessageReactions: FC<MessageReactionsProps> = ({
         onClick={handleMsgActionClick}
         onKeyDown={handleMsgActionKeyDown}
       >
-        <EmojiImg
-          emojiUrl={thumbsUpEmojiUrl}
-          emojiName={t('accessibility.messageActionsMenuThumbsUp')}
-          emojiImgSize={actionMenuEmojiSize}
-        />
+        <EmojiChar emoji={thumbsUpEmoji} size="medium" />
```

Comment:
The menu's size is fixed and not affected by font size choice, the emoji's size should be fixed here.

What about adding "fixed" to the `EmojiSize` type and give it a 16px value?